js/build.js

```diff
@@ -10,9 +10,16 @@ Fliplet.Widget.instance('chart-pie-1-1-0', function(data) {
   if (themeInstance) {
     Object.assign({}, themeInstance.data.values);
 
+    if(!themeValues) {
```

Comment:
Isn't this always undefined if it is only declared as `undefined` 5 lines above?

Reply:
@tonytlwu, first time it's undefined that is why assigning values [here](https://github.com/NikitaJadhavXoriant/fliplet-widget-chart-pie/blob/%40nikitaxor/FIX-ID-3220/js/build.js#L17) but once [setThemeValues()](https://github.com/Fliplet/fliplet-widget-chart-pie/blob/master/js/build.js#L294) method called, then we are getting themeValues.